donations/api.py

```
@@ -0,0 +1,70 @@
+                        },
+                        response_only=True,
+                    ),
+                ],
+            ),
+            500: OpenApiResponse(description="Internal server error"),
+        },
+    )
+    @method_decorator(cache_page(60 * 5))
+    def get(self, request: Request, *args, **kwargs):
+        url = f"{settings.FASTNEAR_RPC_URL}/account/{DONATE_CONTRACT}/view/get_config"
+        response = requests.get(url)
```

Ruff (0.14.1)

Probable use of `requests` call without timeout

(S113)